.circleci/config.yml

```diff
@@ -150,11 +150,13 @@ jobs:
   marvin_tests:
     docker:
       - image: itamararjuan/golang_with_node_aws
+    environment:
+      MARVIN_URL: "http://ec2-34-222-245-15.us-west-2.compute.amazonaws.com:4567"
```

Comment:
why is it here and named thus if you define a different var `MARVIN_ORCHESTRATOR_URL` at the project level?

Reply:
Mistake - removed from config.yml.